src/core/useEvent.ts

```diff
@@ -10,7 +10,7 @@ import { useEffect } from "react";
  * @param callback Callback to call on a new event
  */
 export function useEvent<D>(
-  channel: Channel | PresenceChannel | undefined,
+  channel: Channel | Channel[] | PresenceChannel | undefined,
```

Comment:
Do we want to allow multiple `PresenceChannels` as well?

Reply:
Yes, I just wasn't sure if there were any special considerations. Added to the typing.